config/nightwatch.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   custom_commands_path: './test/util/nightwatch-commands',
   live_output: true,
   parallel_process_delay: 10,
-  disable_colors: false,
+  disable_colors: true,
```

Comment:
It's probably asking too much for this to only be true on Jenkins, eh? So...like if the buildtype is production or some such.